src/main/java/org/fcrepo/spec/testsuite/TestSuiteGlobals.java

```diff
@@ -41,7 +41,7 @@
 public abstract class TestSuiteGlobals {
     public static String cssReport = "reportStyle.css";
     public static String outputDirectory = "report";
-    public static String outputName = "testsuite";
```

Comment:
Is there a reason for changing the output report name?

Reply:
I think this must have been updated automatically when I changed the artifact name from testSuite to testsuite.  I can change this back.

Reply:
I will change this back.